libs/ledger-live-common/src/families/bitcoin/descriptor.ts

```diff
@@ -23,6 +23,7 @@ const perDerivation: Partial<
   "": (fragment) => `pkh(${fragment})`,
   segwit: (fragment) => `sh(wpkh(${fragment}))`,
   native_segwit: (fragment) => `wpkh(${fragment})`,
+  taproot: (fragment) => `tr(${fragment})`,
```

Comment:
Is this what fixes LIVE-2208?

Reply:
yes, there are problems of taproot account when we use satstack with Ledger live because of taproot account descriptor missing